Sources/JavaScriptKit/JSBridgedType.swift

```diff
@@ -0,0 +1,41 @@
+/// Use this protocol when your type has no single JavaScript class.
+/// For example, a union type of multiple classes or primitive values.
```

Comment:
Could this comment give an example of such type?

Reply:
Sure. Would it be OK to link to DOMKit?￼

Reply:
(Here’s an example: https://github.com/swiftwasm/DOMKit/blob/main/Sources/DOMKit/WebIDL/FileOrString.swift)

Reply:
Yes, please do 👍